__tests__/app.test.js

```diff
@@ -45,4 +45,45 @@ describe('GET /api', ()=>{
                 expect(endpoints).toEqual(apiEndpointsJSON)
             })
     })
-})
\ No newline at end of file
+})
+
+describe('GET /api/articles/:articleID', ()=>{
+    test('returns object with correct properties', ()=>{
+        return request(app)
+            .get('/api/articles/2')
+            .expect(200)
+            .then(({body})=>{
+                const {articles} = body
+                expect(articles.length===1).toBe(true)
+                const article = articles[0]
+                expect(article.article_id).toBe(2)
+
+                expect(article).toMatchObject({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        body: expect.any(String),
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String)
+                    })
+                })
+            })
+        test("should return error for valid but non-existent article id", ()=>{
+            return request(app)
+            .get('/api/articles/1000')
+            .expect(404)
+            .then(({body})=>{
+                expect(body.msg).toBe("Not found")
+            })
+        })
+        test("should return error for invalid article id", ()=>{
+            return request(app)
+            .get('/api/articles/notanid')
+            .expect(400)
+            .then(({body})=>{
+                expect(body.msg).toBe('Bad request')
+            })
+        })
+            
+    })
```

Comment:
Good testing overall - just make sure to amend `const article = articles[0]`, as this won't be required if you just send back an object